docs/introduction/1-welcome.mdx

```diff
@@ -23,35 +23,35 @@ import { Meta, Image, Intro, Teaser } from '../../.storybook/components';
 
 Set up a new app with Circuit UI or add it to an existing project.
 
-[Get started](Introduction/Getting-Started/Page)
+[Get started →](Introduction/Getting-Started/Docs)
```

Comment:
"get started rightwards arrow, link"

(this is fine here, in prod I would have suggested putting it in ::after 😛)

Reply:
Excellent point. Addressed in 63029059.